### PR TITLE
wc slider

### PR DESCRIPTION
**Pull request by christophsaile**
created a basic preview of a lory slider

### CODE REVIEW

File: src/components/WcSlider/WcSlider.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+	static attributes = [];
+
+	public methods: WcSliderMethods = {};
+
+	public connectedCallback() {
+		const slider = this.shadowRoot.querySelector('.js_slider');
+		lory(slider, {
+			infinite:1,
+		});
```
Comment (no-creative-name) [inline src/components/WcSlider/WcSlider.ts]: In another project context we had to add a setTimeout of some hundred ms for the initialization to work. But as long as it works without, it's fine! 🥳

File: src/components/WcSlider/styles.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
+@import 'node_modules/@biotope/element/lib/host.mixin';
+
```
Comment (Benschodie) [inline src/components/WcSlider/styles.scss]: Bitte für gesamtes file sass verwenden also zb:
prev {
    xyz: xyz
    next {
        bla: blub

File: src/components/WcSlider/styles.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		min-height: 30vh;
+		width: 880px;
+		margin: 0 auto;
+		position: relative;
+		font-size: 3rem;
+		text-align: center;
+		line-height: 0;
+		overflow: hidden;
+		white-space: nowrap;
+		background-color: lightcoral;
```
Comment (Benschodie) [inline src/components/WcSlider/styles.scss]: Farben bitte in Variablen oberhalb des files

File: src/components/WcSlider/styles.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+	}
+
+	.slides {
+		display: inline-block;
+	}
+
+	li {
```
Comment (no-creative-name) [inline src/components/WcSlider/styles.scss]: Class selectors are more specific and thus less error-prone than HTML element selectors. So I'd prefer to select the items by '.js_slide'.

File: src/components/WcSlider/styles.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+	.prev {
+		left: 0;
+	}
+
+	.next svg,
+	.prev svg {
+		width: 25px;
+	}
+} //END INCLUCDE
```
Comment (no-creative-name) [inline src/components/WcSlider/styles.scss]: Some devs consider it a good practice to order the CSS properties by how much they effect they have. See this a reference: https://webdesign.tutsplus.com/articles/outside-in-ordering-css-properties-by-importance--cms-21685

**Comment by christophsaile** [comment]
close #9

**Pull request merged**